CONTRIBUTING.md

```diff
@@ -2,12 +2,12 @@ This is an adapted version from Contributing.md file used by Community-Committee
 
 ## Vocabulary
 
-* A **Contributor** is any individual creating or commenting on an issue or pull request.
-* A **Collaborator** is a contributor who has been given write access to the repository
+* A **Contributor** is any individual gathering data, performing outreach, feedback collection, organizing meetings, organizing events, taking notes or providing written feedback. As much as possible, we try to capture attribution of these actions with an [issue](https://github.com/nodejs/user-feedback/issues) or a [pull request](https://github.com/nodejs/user-feedback/pulls).
+* A **Collaborator** is a **_Contributor_** who has been given write access to the repository.
+* A **Member** is a **_Collaborator_** with voting rights who has met the requirements of participation to be considered for acceptance, and subsequently voted in by the Node.js CommComm voting process.
```

Comment:
Is this saying "voted in" to the user-feedback group using the same rules/process that CommComm has...  or "voted in" to CommComm?

Reply:
s/voted in by/voted in using